### ISSUE TITLE
double yield statement in sayAll handler

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#6685]
I'm not sure if this is intentional, but it seems to be a bug.
Line 176 sayAllHandler.py
```
index=lastSentIndex+1
		speech.speak([speech.IndexCommand(index)])
		while speech.getLastSpeechIndex()<index:
			yield
			yield
		# Some synths say they've handled the index slightly sooner than they actually have,
```

**Comment by jcsteh** [comment]
It's intentional, if somewhat arbitrary. Those generators run every core tick (which is every 10 ms or so while a core generator exists) and there's no reason for say all to do stuff that often. Thus, we do a few extra yields to slow it down. This kind of sucks and is the last remaining chunk of core spinning. This would be far better handled by speech callbacks (#4877) when we get those.

We should still probably comment this, even though that code will be replaced eventually.

**Comment by leonardder** [comment]
Fixed in #7599
